build.gradle.kts

```diff
@@ -65,6 +65,20 @@ val tools = compiler.get().metadata.installationPath.file("lib/tools.jar")
 dependencies {
     compileOnly(files(tools))
     compileOnly("org.jetbrains:annotations:24.0.+")
+
+    // Needed for annotation processing tests
+    testImplementation(files(tools))
+    testImplementation("org.openrewrite:rewrite-java:latest.release")
```

Comment:
I wonder if we should use `latest.integration` instead?

Reply:
Done! Didn't think it would be necessary often, but best to be consistent I guess. We'll see if nebula picks things up correctly on the next release.